main.go

```diff
@@ -45,8 +43,7 @@ func main() {
 
 	// Write log to file if the log flag was provided
 	if flags.Log != "" {
-		logFile, err := os.OpenFile(flags.Log, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
-		if err != nil {
+		if logFile, err := os.OpenFile(flags.Log, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666); err != nil {
```

Comment:
Ah my mistake, I didn't notice logFile not being defined to begin with
I'll edit the code in a bit